databroker/src/grpc/kuksa_val_v1/val.rs

```diff
@@ -344,6 +279,142 @@ impl proto::val_server::Val for broker::DataBroker {
         }))
     }
 
+    type StreamedUpdateStream =
+        ReceiverStream<Result<proto::StreamedUpdateResponse, tonic::Status>>;
+
+    async fn streamed_update(
+        &self,
+        request: tonic::Request<Streaming<proto::StreamedUpdateRequest>>,
+    ) -> Result<tonic::Response<Self::StreamedUpdateStream>, tonic::Status> {
+        debug!(?request);
+        let permissions = match request.extensions().get::<Permissions>() {
+            Some(permissions) => {
+                debug!(?permissions);
+                permissions.clone()
+            }
+            None => return Err(tonic::Status::unauthenticated("Unauthenticated")),
+        };
+        let mut stream = request.into_inner();
+
+        let mut shutdown_trigger = self.get_shutdown_trigger();
+
+        // Copy (to move into task below)
+        let broker = self.clone();
+
+        // Create stream (to be returned)
+        let (sender, receiver) = mpsc::channel(10);
```

Comment:
Should we possibly have a comment here explaining why we have selected the magic number 10. Why not 42?

Reply:
I took it over from here: https://github.com/eclipse-kuksa/kuksa-databroker/blob/main/databroker/src/grpc/sdv_databroker_v1/collector.rs#L106 

I have the feeling that value is set arbitrarily and can be set freely to 42 or even 100. The example code of channel goes with a "default" of 100. Shall I increase the value or just add a comment, that it's value has been set arbitrarily?

Reply:
Since it is a buffer https://docs.rs/tokio/latest/tokio/sync/mpsc/fn.channel.html we should think about this. With this value after buffering 10 messages it will put any more messages on the channel and will wait.

Reply:
I assume the message size is not that big so having 100 is not a real problem, or? Anyway, whatever number we come up with I think it could be good to mention that it is an just an educated guess and not based on any real tests.

Reply:
I have a vague memory of measuring the throughput given different buffer sizes and that is how I ended up with a channel of size 10.

I guess a comment would have been a good thing to add then, but at the same time I don't think the testing was very thorough nor sound. But if we are to change it, we should probably do it while measuring the resulting behaviour.

Reply:
Do we think this is a number that better should be a config (in the long term)? We previously had discussions on constants for max number of signals, max number of subscriptions and so on. Is something were think tweaking might be necessary for different deployment based on memory and traffic. I.e. in one system maybe 10 is a good choice as traffic never is high and memory scarce, in another it is the opposite. Or do we think we can have one number that fits all?

Reply:
Probably a comment owuld be good. Whether such details need to be configurable: I am unsure, if this is a "user-serviceable setting" it would also imply more testing.

Anyway, lets not "solve" this in this PR